python/eos/data/native.py

```diff
@@ -526,7 +527,10 @@ def __init__(self, path):
         f = os.path.join(path, 'dynesty_results.npy')
         if not os.path.exists(f) or not os.path.isfile(f):
             raise RuntimeError('Dynesty results file {} does not exist or is not a file'.format(f))
-        self.results = _np.load(f)
+        
+        load_res = _np.load(f, allow_pickle=True)
+        dict_res = load_res.item()
+        self.results = dynesty.results.Results(dict_res)
```

Comment:
Since the two commits deal with loading the `dynesty` results, merge them please.

Reply:
Yes